UHResidentInformationAPI/V1/Factories/ResponseFactory.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using UHResidentInformationAPI.V1.Boundary.Responses;
+using Address = UHResidentInformationAPI.V1.Domain.Address;
+using AddressResponse = UHResidentInformationAPI.V1.Boundary.Responses.Address;
+using ResidentInformation = UHResidentInformationAPI.V1.Domain.ResidentInformation;
+using ResidentInformationResponse = UHResidentInformationAPI.V1.Boundary.Responses.ResidentInformation;
+
+namespace UHResidentInformationAPI.V1.Factories
+{
+    public static class ResponseFactory
+    {
+        public static ResidentInformationResponse ToResponse(this ResidentInformation domain)
+        {
+            return new ResidentInformationResponse
+            {
+                HouseReference = domain.HouseReference,
+                PersonNumber = domain.PersonNumber,
+                FirstName = domain.FirstName,
+                LastName = domain.LastName,
+                DateOfBirth = domain.DateOfBirth,
+                NINumber = domain.NINumber,
+                PhoneNumber = domain.PhoneNumber.ToResponse(),
+                Email = domain.Email.ToResponse(),
+                Address = domain.Address.ToResponse()
```

Comment:
Will these 3 fields always be there? If any of them is null this will throw a null value exception

Reply:
Had a look at the schema: values for the email address and the phone number can't be null so these would probably always exist but the address and postcode could be null.

Will add code to account for this. Thanks 👍

Reply:
Changes made in 6525ce07992b8187a2122491d6f4d5173ef5db40